server/index.js

```diff
@@ -1,11 +1,19 @@
-const serve = require('koa-static');
 const Koa = require('koa');
+const serve = require('koa-static');
+const { get } = require('koa-route');
 
 const app = new Koa();
 
-// $ GET /hello.txt
+const data = require('../src/data/summits.json');
+
+// JSON endpoint, adapted from https://github.com/koajs/route
+app.use(
+  get('/api/v1/peaks', ctx => {
+    ctx.body = data;
+  })
+);
```

Comment:
Another note on arrow functions: These two are the same:

```js
app.use(get( … )) // implicit return: get() passes its value to app.use()
```

```js
app.use(() => {
  return get( … ) // explicit return through a function: get() passes its value to our anonymous function, which then passes its value to app.use()
}
```

In #2, we’re inserting an extra function, but we’re returning the value, so it’s the same. #2 would be pretty dumb—it doesn’t do anything—but it may be helpful to think you always have this option.

`get( … )` is built to return a value, which needs to get returned somewhere that uses it.

---

It may be helpful to ask yourself for every function: _what is this returning_?

Take the internal function:

```js
  get('/api/v1/peaks', ctx => {
    ctx.body = data;
  })
```

We’re actually calling a function for the 2nd parameter. That’s the same as this:

```js
  get('/api/v1/peaks', function (ctx) {
    ctx.body = data;
  })
```

The only difference is that the arrow function just saves us space (there are other minor differences, but don’t worry about that now). Notice that **we’re not returning anything.** Now, to me, this is weird. But I’m just following the examples so I assume everything works all right, and this PR works. So we can assume that that `ctx` value—even though we’re not returning it—is tracked and taken care of within our `koa-route` package perfectly. So we can just not think about it!

But anyways, the point is to just focus on `return` statements—explicit or implicit. Not sure what’s returning? `console.log` it! If it’s returning nothing, it’ll show `undefined`.